ember-core/shared/src/main/scala/org/http4s/ember/core/h2/H2Server.scala

```diff
@@ -194,8 +195,10 @@ private[ember] object H2Server {
       F.sleep(1.seconds) >> stateRef.get.map(_.closed).ifM(F.unit, holdWhileOpen(stateRef))
 
     def initH2Connection: F[H2Connection[F]] = for {
-      address <- socket.remoteAddress
-      (remotehost, remoteport) = (address.host, address.port)
+      address <- socket.remoteAddress.attempt.map(
+        // TODO, only used for logging
+        _.leftMap(_ => UnixSocketAddress("unknown.sock"))
+      )
```

Comment:
Yeah, I'm sort of flaking on this. /shrug

Reply:
Hmmmm. This is definitely weird.

Reply:
😮

Reply:
Yeah, so for the record the upstream issue is here.
- https://github.com/Comcast/ip4s/issues/466